app/views/services/govbox/pricing.html.erb

```diff
@@ -31,12 +31,42 @@
           </ol>
         </li>
       </ul>
+
+      <h1 class="mt-5">Cenník GovBox API</h1>
+      <p class="mb-5"><small>Platný od 28.9.2019</small></p>
+
+      <p>Poplatok za poskytnutie služby GovBox API (odmena) sa stanovuje na základe počtu Používateľom stiahnutých elektronických správ (definovaných v <%= link_to 'Podmienkach používania služby GovBox API', terms_services_govbox_index_path %>) za podmienok uvedených v Zmluve a v Podmienkach používania služby GovBox API, pričom</p>
+      <ul>
+        <li>prvý mesiac nasledujúci po mesiaci v ktorom Zmluva nadobudla účinnosť je služba GovBox API poskytovaná bezplatne; a v nasledujúcich mesiacoch je služba účtovaná na základe kategórie, do ktorej spadá, a to podľa počtu stiahnutých elektronických správ nasledovne:
+          <ol type="A">
+            <li><span>v prípade ak v kalendárnom mesiaci nebude stiahnutá žiadna elektronická správa, tak je služba za tento mesiac poskytovaná bezodplatne;</span></li>
+            <li><span>pri počte elektronických správ od 1 do 20 stiahnutých elektronických správ za kalendárny mesiac predstavuje odmena Poskytovateľa 8,00 EUR / mesačne;</span></li>
+            <li><span>pri počte elektronických správ od 21 do 50 stiahnutých elektronických správ za kalendárny mesiac predstavuje odmena Poskytovateľa 16,00 EUR / mesačne;</span></li>
+            <li><span>pri počte elektronických správ od 51 do 100 stiahnutých elektronických správ za kalendárny mesiac predstavuje odmena Poskytovateľa 32,00 EUR / mesačne;</span></li>
+            <li><span>pri počte elektronických správ od 101 do 500 stiahnutých elektronických správ za kalendárny mesiac predstavuje odmena Poskytovateľa 64,00 EUR / mesačne;</span></li>
+            <li><span>pri počte elektronických správ vyššom ako 500 stiahnutých elektronických správ za kalendárny mesiac je odmena Poskytovateľa predmetom samostatnej dohody medzi Poskytovateľom a Používateľom a do času uzatvorenia dohody sa použije výpočet uvedený v Zmluve.</span></li>
+          </ol>
+        </li>
+        <li>Integračný poplatok je predmetom dohody Používateľa a Poskytovateľa a jeho výška je uvedená v Zmluve.</li>
+        <li>
+          Ak je Používateľ zároveň členom občianskeho združenia Slovensko.Digital, IČO: 50158635, tak po dobu trvania členstva Používateľa v občianskom združení bude odmena za poskytnutie služby GovBox účtovaná podľa kategórie členstva v tomto združení nasledovne:
+          <ol type="A" start="7">
```

Comment:
@jsuchal @smith558  preco je tu `start="7"`? mne to cislovanie takto pride dako divne: https://ekosystem.slovensko.digital/sluzby/govbox/cennik

Reply:
teraz ako to je to berieme ako jeden rozdeleny `ol` , s tym predelom v strede a potom pokracuje od` 7`, to jest `G` . Semanticky mi to tiez pride cudne kedze vyznamovo by to mali byt 2 independet `ol `. Prebral som to  predosleho dizajnu ako to bolo

![image](https://user-images.githubusercontent.com/44023416/64913429-5b3b4300-d740-11e9-8d25-651dbcd9a95d.png)

Reply:
Ja by som bol aj za takyto `ul `skor

![image](https://user-images.githubusercontent.com/44023416/64913467-f92f0d80-d740-11e9-95bc-498e3074e828.png)

Reply:
Asi pravnici mysleli, ze sa cez tie pismena budeme odvolavat na tie casti ale uznavam, ze to nedava uplne zmysel lebo sa to tam nikde nespomina.

Reply:
@smith558 dajme tam teda `ul` (ale unstyled, bez medzery hore ako mas v screene)

Reply:
mal som chvilku, tak som to pozrel, tie pismena sa tam spominaju, upravil som to jemne aby to vyzeralo ako pokracujuci zoznam (po dvojbodke v odrazkach to vzdy zacne malym pismenom a skonci ciarkou / bodkou na uplnom konci)

https://github.com/slovensko-digital/www.ecosystem/commit/6f57291f8744970973e386a1928dfd4641a8d51a